examples/cloud-run/variables.tf

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  type        = string
+  description = "The project ID to manage the Secret Manager resources."
+}
+
+variable "region" {
+  type        = string
+  description = "The region which the resources will be created at."
+  default     = "us-central1"
+}
+
+variable "keyring" {
+  description = "KMS Keyring name."
+  type        = string
+}
+
+variable "key" {
+  description = "KMS Key name."
+  type        = string
+}
```

Comment:
Since we are creating these every time, how about we have default values here, so users can run the example easily?